File: sort/javascript/bubbleSort.js
function bubbleSort(arr) {
  for (let i = 0; i < arr.length - 1; i++) {
    for (let j = 0; j < arr.length - i - 1; j++) {
      if (arr[j] > arr[j + 1]) {
        const tmp = arr[j];
        arr[j] = arr[j + 1];
        arr[j + 1] = tmp;
      }
    }
  }
  return arr;
}

const arr = [2, 6, 4, 3, 10, 9, 1, 5, 7, 8];
console.log(`Unsorted: ${arr}`);
console.log(`Sorted with Bubble Sort: ${bubbleSort(arr)}`);

File: sort/python/bubbleSort.py
def bubbleSort(arr):
    for i in range(len(arr) - 1):
        for j in range(len(arr) - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

    return arr


arr = [2, 6, 4, 3, 10, 9, 1, 5, 7, 8]
print("Unsorted:", arr)
print("Bubble Sort:", bubbleSort(arr))
